### PR TITLE
feat: switch back to protobufjs

### PR DESCRIPTION
**Pull request by tuyennhv**
**Motivation**
- protons is too slow to use, see https://github.com/ipfs/protons/issues/51

**Description**
- Switch back to protobufjs
- Copy generated files from v1.0.0
- Change `rpc.js` to `rpc.cjs`, new `rpc.js` to reexport `RPC` from `rpc.js`

### CODE REVIEW

**Comment by mpetrunic** [comment]
@tuyennhv have you tried with new versions of protons?https://github.com/ipfs/protons/releases/tag/protons-v4.0.1
https://github.com/ipfs/protons/compare/protons-runtime-v2.0.1...protons-runtime-v2.0.2

should have performance optimizations

File: src/message/index.ts [hunk 1]
```diff
@@ -1 +0,0 @@
-export * from './rpc.js'
```
Comment (wemeetagain) [inline src/message/index.ts:1]: We still need this file for the subpath export (`import ... from '@chainsafe/libp2p-gossipsub/message'`)

Comment (tuyennhv) [reply src/message/index.ts:1]: yeah this is exactly what I got when I build lodestar 👍

**Comment by tuyennhv** [comment]
> @tuyennhv have you tried with new versions of protons?https://github.com/ipfs/protons/releases/tag/protons-v4.0.1
https://github.com/ipfs/protons/compare/protons-runtime-v2.0.1...protons-runtime-v2.0.2

@mpetrunic I've tried it, below is the result:
- current protons
```
rpc
    ✔ decode Attestation message                                           7089685 ops/s    141.0500 ns/op        -    2045205 runs   30.1 s
```
- new protons (v4.0) & protons-runtimes (v2.0)
```
rpc
    ✔ decode Attestation message                                           6807815 ops/s    146.8900 ns/op        -    1977187 runs   30.1 s
```
- protobufjs
```
rpc
    ✔ decode Attestation message using protobufjs                      1.315789e+8 ops/s    7.600000 ns/op        -   28030718 runs   30.0 s
```

=> there is no improvement made with new protons and it's still 20x slower than protobufjs

**Review by dapplion — APPROVED** [review]
LGTM!

**Review by wemeetagain — APPROVED** [review]

**Comment by achingbrain** [comment]
> there is no improvement made with new protons and it's still 20x slower than protobufjs

Can you please share the benchmarks you're using to make these measurements?

**Comment by tuyennhv** [comment]
> > there is no improvement made with new protons and it's still 20x slower than protobufjs
> 
> Can you please share the benchmarks you're using to make these measurements?

@achingbrain the benchmark is in https://github.com/ChainSafe/js-libp2p-gossipsub/issues/318

**Pull request merged**